src/storage/FCS.cpp

```diff
@@ -432,7 +445,7 @@ bool FB_Storage::fcs_sendRequest(FirebaseData *fbdo, struct fb_esp_fcs_req_t *re
     else
         fbdo->_ss.connected = false;
 
-    if (Signer.getCfg()->_int.fb_file && req->requestType == fb_esp_fcs_request_type_download)
+    if (Signer.getCfg()->_int.fb_file && req->requestType == fb_esp_fcs_request_type_download && req->storageType != mem_storage_type_ota_update)

```

Comment:
It can be
`Signer.getCfg()->_int.fb_file && req->requestType == fb_esp_fcs_request_type_download)`

As we have request type `fb_esp_fcs_request_type_download_ota`